### ISSUE TITLE
no log message for over 2 days with verbose 4

### ISSUE THREAD

**Issue by xNxExOx** [cmu-sei/pharos#99]
- input file: [BattleForge.exe](https://drive.google.com/file/d/1wi-PrvEtRBGVO8desjj5IESIuYEfvpQJ/view?usp=sharing)
- command `ooanalyzer -f /dir/BattleForge.exe --partitioner-timeout 999999 --per-function-timeout=600 --timing -j /dir/both.json --threads=40 --maximum-memory 200704 --serialize=/dir/BatteForge.both.serialize --prolog-facts=BatteForge.both.facts.P --verbose=4`
- result: https://cdn.discordapp.com/attachments/562879368083275789/719270007011409990/unknown.png

Did we make a mistake by pressing `CTRL+C` after more than 2 days without any activity?
Or is it another bug related to this input file?
I would expect at least some logs every hour or so so the user know that it is still running, because this way we have no evidence if it got stuck on something, or still working :( so after more than two days (as you can see from the "result" image) we pressed "CTRL+C and it ended.

**Comment by sei-eschwartz** [comment]
Your file is one of the largest we've ever tried with OOAnalyzer.  It's not going to work using XSB.  Please follow the instructions in https://github.com/cmu-sei/pharos/blob/master/share/prolog/oorules/README.md starting at step three.

**Comment by FallenAvatar** [comment]
Just want to let @xNxExOx that I am also analyzing a large game executeable (18MB - Firefall) And I ran into an infinite recursion (and/or out of stack space?) error. (Error message was unclear)

I edited the oodebugrun-swipl file to increase the max stack space (from 20GB -> 30GB) and just started another run. So just wanted to let you know that you might need to do something similar.

If it errors out again, I will be posting another issue about it.
